importScripts/sampleRecords/locationHistory.destination.json

```diff
@@ -164,166 +164,146 @@
     {
       "movement_id":"11",
       "id":"11",
-      "idno":"108",
+      "idno":"120",
       "display_label":"???",
       "ca_movements.type_id":"Movement",
-      "ca_movements.idno":"108",
+      "ca_movements.idno":"120",
       "ca_movements.AuthorizedBy":"Curator",
       "ca_movements.MovementDate":"4 November 1998",
-      "ca_movements.LocationStatus":"Stored",
-      "ca_objects":"A1937.46",
+      "ca_movements.LocationStatus":"Displayed",
+      "ca_objects":"A1949.11o",
       "ca_objects.type_id":"Artwork",
-      "locationHierarchy":"Locked Storeroom;B4",
-      "ca_storage_locations":"B4",
-      "ca_storage_locations.type_id":"container"
+      "locationHierarchy":"Hall;North Wall",
+      "ca_storage_locations":"North Wall",
+      "ca_storage_locations.type_id":"shelf"
     },
     {
       "movement_id":"12",
       "id":"12",
-      "idno":"109",
+      "idno":"155",
```

Comment:
oh dear. git has gotten confused as to the order of records, as we're now excluding historical records where the storage location no longer exists (at request of the client) and have also added a number of new sample records. From here on down the records on the left do not correspond to the records on the right.